src/sumo/wrapper/sumo_client.py

```diff
@@ -378,3 +379,21 @@ def get_aggregate(self, json: dict):
             Sumo aggregate response object
         """
         return self.agg_client.get_aggregate(json)
+
+    def getLogger(self, name):
+        """Gets a logger object that sends log objects into the message_log
+        index for the Sumo instance.
+
+        Args:
+            name: string naming the logger instance
+
+        Returns:
+            logger instance
+
+        See Python documentation for logging.Logger for details.
+        """
+
+        logger = logging.getLogger(name)
+        handler = LogHandlerSumo(self)
+        logger.addHandler(handler)
+        return logger
```

Comment:
so this will make it an option to send messages to the message-log, but we are not enabling it by default?

Reply:
Right. In order to log anything to Sumo, a module would have to do something like
```
logger = sumoclient.getLogger("sumo.application.something")
logger.setLevel(logging.INFO) # optional; default is logger.WARNING
...
logger.info("hello, %s!", "world")
```